packages/web/src/components/toolbarButton.css

```diff
@@ -16,15 +16,25 @@
 
 .toolbar-button {
   flex: none;
-  border: none;
-  outline: none;
   color: var(--vscode-sideBarTitle-foreground);
   background: transparent;
-  padding: 4px;
   margin-left: 10px;
-  cursor: pointer;
   display: inline-flex;
   align-items: center;
+  border-radius: 6px;
+  border: 1px solid var(--vscode-dropdown-border);
+  font-family: inherit;
+  line-height: 20px;
```

Comment:
We already do flex + align items, why is this needed?

Reply:
removed